### PR TITLE
Appveyor

### PR DESCRIPTION
**Pull request by Anakonda**
appveyor.yml for automatic windows builds. Testing is now disabled as appveyor doesn't upload builds if any of the tests fails (200+ fails if I remember)

Latest build will be https://ci.appveyor.com/api/projects/'account'/'project'/artifacts/'artifact_file_path'?branch='branch'
So this PR is https://ci.appveyor.com/api/projects/anakonda/borg/artifacts/windowsbuild.zip?branch=appveyor

### CODE REVIEW

**Comment by PlasmaPower** [comment]
This is through MYSYS, not Cygwin correct? I think the plan was to use Cygwin, but this might also work.

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+environment:
+  matrix:
+    - compiler: msys2
+      MSYS2_ARCH: x86_64
+      MSYS2_DIR: msys64
+      MINGW: MINGW64
+install:
+  - 'SET PATH=C:\%MSYS2_DIR%\usr\bin;C:\%MSYS2_DIR%\%MINGW%\bin;%PATH%'
+  - 'bash -lc "pacman --noprogressbar --noconfirm --needed -S mingw-w64-%MSYS2_ARCH%-{python3,gcc,lz4,python3-pip,cython,ntldd-git} git openssl"'
+  - 'bash -lc "python3 -m pip install -U pip"'
```
Comment (PlasmaPower) [inline appveyor.yml]: Why the `-l`?

File: appveyor.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+build_script:
+  - 'pip3 install -e .'
+  - 'python3 deployment/windows/buildwin32.py'
+test_script:
+  - 'echo tests disabled'
+  # - 'py.test --cov=borg --cov-config=.coveragerc --benchmark-skip --pyargs src/borg/testsuite'
+artifacts:
+  - path: win32exe
+    name: windowsbuild
+    type: zip
```
Comment (PlasmaPower) [inline appveyor.yml]: Could you add a newline at the end of this file?

**Comment by Anakonda** [comment]
Correct. The plan was to use Cygwin when borg didnt run on non-Cygwin.

File: appveyor.yml (hunk 1 above)
Comment (Anakonda) [inline appveyor.yml]: Makes it run '/etc/profile' first. Some important stuff there.

**Comment by ThomasWaldmann** [comment]
https://ci.appveyor.com/project/ThomasWaldmann/borg \o/

Somehow it would be better under /project/borgbackup/borg though.

**Comment by ThomasWaldmann** [comment]
OK, any idea how we can increase test coverage from 0?

**Comment by PlasmaPower** [comment]
@ThomasWaldmann Maybe make some tests OS dependent?

**Pull request merged**